client/dashboard/sites/performance/index.tsx

```diff
@@ -63,28 +67,30 @@ function SitePerformanceContent( { site }: { site: Site } ) {
 		return null;
 	}
 
+	const isLoading =
+		isLoadingPages || isFetchingReport || isRunningDesktopReport || isRunningMobileReport;
+
 	return (
 		<PageLayout>
-			<PageHeader
-				title={ __( 'Performance' ) }
-				description={
-					<Button isPrimary onClick={ handleReportRefetch }>
-						Retest
-					</Button>
-				}
-				actions={
-					<>
-						<PageSelectorWrapper
-							siteUrl={ site.URL }
-							currentPage={ currentPage }
-							pages={ pagesData }
-							onChange={ handlePageChange }
-							disabled={ isFetchingReport || isRunningDesktopReport || isRunningMobileReport }
-						/>
-						<DeviceTabControls value={ deviceToggle } onChange={ setDeviceToggle } />
-					</>
-				}
-			/>
+			<PageHeader title={ __( 'Performance' ) } />
+			<HStack spacing={ 2 } alignment="flex-end" justify="flex-start" expanded={ false }>
```

Comment:
@p-jackson I'm curious why this changed to using the `<HStack>`? I think we should still be passing components into <[PageHeader](https://github.com/Automattic/wp-calypso/pull/103159/files#diff-e7dff45adc51937d55a458a793e7e0986032e721fcbea258d18ac80cbe801a09R3)>....?